docs-translations/ko-KR/api/session.md

```diff
@@ -53,7 +53,7 @@ const ses = session.fromPartition('persist:name');
 
 `Session` 객체는 다음과 같은 이벤트를 가지고 있습니다:
 
-#### Event: 'will-download'
+### Event: 'will-download'
```

Comment:
The `Event: 'will-download'` belongs to `Instance Events` section, so I think we should use `####` here?